tests/manage/z_cluster/nodes/test_nodes_maintenance.py

```diff
@@ -484,6 +489,16 @@ def test_pdb_check_simultaneous_node_drains(
         time.sleep(30)
         try:
             drain_nodes([node_B])
+            # Check Mon failovers in 10mins, 15mins and 20 mins
+            timeout = [600, 300, 300]
+            for failover in timeout:
+                sample = TimeoutSampler(
+                    timeout=failover,
+                    sleep=10,
+                    func=helpers.check_mon_pods_eq_3,
+                )
+                if sample.wait_for_func_status(result=True):
+                    assert "4rth mon is found in the cluster"
```

Comment:
Please correct typo here, it should 4th.
How about changing it to "There are more than 3 mon pods" to make the statement more generic

Reply:
Done